app/assets/stylesheets/media/dashicons.scss.erb

```diff
@@ -0,0 +1,1026 @@
+	content: "\f213";
+}
+
+.dashicons-editor-justify:before {
+	content: "\f214";
+}
+
+.dashicons-editor-textcolor:before {
+	content: "\f215";
+}
+
+.dashicons-editor-paste-word:before {
+	content: "\f216";
+}
+
+.dashicons-editor-paste-text:before {
+	content: "\f217";
+}
+
+.dashicons-editor-removeformatting:before {
+	content: "\f218";
+}
+
+.dashicons-editor-video:before {
+	content: "\f219";
+}
+
+.dashicons-editor-customchar:before {
+	content: "\f220";
+}
+
+.dashicons-editor-outdent:before {
+	content: "\f221";
+}
+
+.dashicons-editor-indent:before {
+	content: "\f222";
+}
+
+.dashicons-editor-help:before {
+	content: "\f223";
+}
+
+.dashicons-editor-strikethrough:before {
+	content: "\f224";
+}
+
+.dashicons-editor-unlink:before {
+	content: "\f225";
+}
+
+.dashicons-editor-rtl:before {
+	content: "\f320";
+}
+
+.dashicons-editor-break:before {
+	content: "\f474";
+}
+
+.dashicons-editor-code:before {
+	content: "\f475";
+}
+
+.dashicons-editor-paragraph:before {
+	content: "\f476";
+}
+
+.dashicons-editor-table:before {
+	content: "\f535";
+}
+
+/* Post Icons */
+
+.dashicons-align-left:before {
+	content: "\f135";
+}
+
+.dashicons-align-right:before {
+	content: "\f136";
+}
+
+.dashicons-align-center:before {
+	content: "\f134";
+}
+
+.dashicons-align-none:before {
+	content: "\f138";
+}
+
+.dashicons-lock:before {
+	content: "\f160";
+}
+
+.dashicons-unlock:before {
+	content: "\f528";
+}
+
+.dashicons-calendar:before {
+	content: "\f145";
+}
+
+.dashicons-calendar-alt:before {
+	content: "\f508";
+}
+
+.dashicons-visibility:before {
+	content: "\f177";
+}
+
+.dashicons-hidden:before {
+	content: "\f530";
+}
+
+.dashicons-post-status:before {
+	content: "\f173";
+}
+
+.dashicons-edit:before {
+	content: "\f464";
+}
+
+.dashicons-post-trash:before,
+.dashicons-trash:before {
+	content: "\f182";
+}
+
+.dashicons-sticky:before {
+	content: "\f537";
+}
+
+
+/* Sorting */
+
+.dashicons-external:before {
+	content: "\f504";
+}
+
+.dashicons-arrow-up:before {
+	content: "\f142";
+}
+
+.dashicons-arrow-down:before {
+	content: "\f140";
+}
+
+.dashicons-arrow-left:before {
+	content: "\f141";
+}
+
+.dashicons-arrow-right:before {
+	content: "\f139";
+}
+
+.dashicons-arrow-up-alt:before {
+	content: "\f342";
+}
+
+.dashicons-arrow-down-alt:before {
+	content: "\f346";
+}
+
+.dashicons-arrow-left-alt:before {
+	content: "\f340";
+}
+
+.dashicons-arrow-right-alt:before {
+	content: "\f344";
+}
+
+.dashicons-arrow-up-alt2:before {
+	content: "\f343";
+}
+
+.dashicons-arrow-down-alt2:before {
+	content: "\f347";
+}
+
+.dashicons-arrow-left-alt2:before {
+	content: "\f341";
+}
+
+.dashicons-arrow-right-alt2:before {
+	content: "\f345";
+}
+
+.dashicons-leftright:before {
+	content: "\f229";
+}
+
+.dashicons-sort:before {
+	content: "\f156";
+}
+
+.dashicons-randomize:before {
+	content: "\f503";
+}
+
+.dashicons-list-view:before {
+	content: "\f163";
+}
+
+.dashicons-exerpt-view:before, /* Misspelled. Use .dashicons-excerpt-view instead. */
```

Comment:
__[Misspell]__

"exerpt" is a misspelling of "excerpt"


<!-- Comment by [SideCI](https://sideci.com) -->